tools/dynamo/gb_id_mapping.py

```diff
@@ -0,0 +1,146 @@
+import ast
+import json
+import re
+from pathlib import Path
+
+
+def get_source_segment(source, node):
+    if hasattr(ast, "get_source_segment"):
+        return ast.get_source_segment(source, node)
+    else:
+        return source[node.lineno - 1][node.col_offset : node.end_col_offset]
+
+
+def clean_string(s):
+    if s is None:
+        return None
+    if isinstance(s, str):
+        s = re.sub(r'^f["\']', r'"', s)
+        s = re.sub(r'["\'] f["\']', " ", s)
+        s = re.sub(r'^["\'](.*)["\']$', r"\1", s)
+        s = re.sub(r"\s*\n\s*", " ", s)
+        s = s.replace('\\"', '"').replace("\\'", "'")
+        s = s.replace("\\", "")
+        s = re.sub(r'" "', " ", s)
+        return s
+    return s
+
+
+def expand_hints(hints):
+    from torch._dynamo import graph_break_hints
+
+    expanded_hints = []
+    for hint in hints:
+        if isinstance(hint, str):
+            if "*graph_break_hints.USER_ERROR" or "graph_break_hints.USER_ERROR" in hint:
```

Comment:
Iterate on the attributes of `graph_break_hints` rather than hardcoding one-by-one.